pysite/views/api/bot/tag.py

```diff
@@ -1,5 +1,4 @@
 # coding=utf-8
-
```

Comment:
Formatting commits don't really belong in PRs, especially not trivial ones like this

Reply:
I, on the other hand, think trivial formatting commits are okay.

Reply:
this was an accident lol